tensorflow/lite/micro/kernels/xcore/xcore_interpreter.cc

```diff
@@ -26,6 +29,15 @@ XCoreInterpreter::XCoreInterpreter(const tflite::Model* model,
     : tflite::MicroInterpreter(model, resolver, allocator, reporter, profiler),
       dispatcher_(reporter, use_current_thread) {
   SetDispatcher(&dispatcher_);
+
+  // Create and set the reference MemoryLoader
+  //   The MemoryLoader is dynamically allocated from the arena
+  unsigned char* loader_buf = static_cast<unsigned char*>(
+      allocator->AllocatePersistentBuffer(sizeof(MemoryLoader)));
+  MemoryLoader* loader = new (loader_buf) MemoryLoader();
```

Comment:
If `loader_buf` is not used for anything else, the cast is not necessary:
```suggestion
  auto* loader_buf = allocator->AllocatePersistentBuffer(sizeof(MemoryLoader));
  auto* loader = new (loader_buf) MemoryLoader();
```